src/Provider/Provider.Web/Configuration/EmployerRecruitSystemConfiguration.cs

```diff
@@ -0,0 +1,7 @@
+namespace Esfa.Recruit.Provider.Web.Configuration
+{
+    public class ProviderRecruitSystemConfiguration
```

Comment:
Maybe rename the file `ProviderRecruitSystemConfiguration.cs`

Reply:
👍